securedrop/db.py

```diff
@@ -368,26 +368,26 @@ def verify_token(self, token):
                     return True
             return False
 
+    _LOGIN_ATTEMPT_PERIOD = 60  # seconds
+    _MAX_LOGIN_ATTEMPTS_PER_PERIOD = 5
```

Comment:
I think it is slightly nicer to set these as class attributes as you've done. Good call!

**Edit:** I see exactly why now :smile: